README.md

```diff
@@ -1,7 +1,8 @@
 ## C# for Visual Studio Code (powered by OmniSharp)
 
-Don't install this extension unless you want to try an early version. By default it is already
-bundled with Visual Studio Code.
+Don't install this extension unless you want to try an early version. 
```

Comment:
How about something like --
This is a preview of C# support in Visual Studio code. It is designed to work well with upcoming .NET Core 1.0 -- see link for more information on what .NET Core 1.0 is. These new projects are based on project.json and the .NET CLI (links).

For msbuild based projects, or ASP.NET Core 5.0 you still want to use the omnisharp extension instead (link).